spec/models/api_response_spec.rb

```diff
@@ -0,0 +1,133 @@
+require "rails_helper"
+require "fake_issues"
+
+RSpec.describe ApiResponse do
+  context "a successful response" do
+    let(:params) { Net::HTTPSuccess.new(1.1, "200", "OK") }
+    let(:response_body) do
+      {
+        data: {
+          grantee: "Some Grantee"
+        },
+        meta: {
+          itemTotalCount: 20
+        }
+      }
+    end
+    let(:subject) { ApiResponse.new(params) }
```

Comment:
[suggestion (nonblocking)] the `subject` is always setup the same, so I'd prefer pulling it to the top of the describe block.

[nitpick (nonblocking)] Pretty sure `subject` used here is an alias to `let(:subject)` but I like setting the subject with just `subject { ApiResponse.new(params) }`